### PR TITLE
Implemented static code for timeline page(#27xkt1t)

### LINKED ISSUE #121: Create timeline detail view
One mobile, the timeline view on the order detail page should open up in another page instead of inline since there isn't enough space.

Screen Shot 2022-04-19 at 4 12 12 PM

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#121 

Closes #121 

### Short Description and Why It's Useful
One mobile, the timeline view on the order detail page should open up in another page instead of inline since there isn't enough space.

### Screenshots of Visual Changes before/after (If There Are Any)
timeline-page

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/hotwax/commerce-hub#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
https://user-images.githubusercontent.com/82817616/163995113-09a27fd8-9bd3-4def-b4ce-a73c9d46e669.mp4

**Review by dt2patel — APPROVED** [review]
Looks good to me @adityasharma7 merge if this looks good to you.

File: src/views/Order.vue [hunk 1]
```diff
@@ -27,9 +27,13 @@
           </div>
 
           <!-- TODO: Update timeline to display the orderCreatedDate, orderCompletedDate, brokeredDate and entryDate -->
-          <!-- <div class="timeline">
-            <ion-item lines="none">
-              <ion-icon slot="start" :icon="timeOutline" class="mobile-only" />
+          <div class="timeline">
```
Comment (adityasharma7) [inline src/views/Order.vue:30]: Use Timeline component here

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Order.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Order.vue:30]: I have created a timeline detail component and used it on the order, purchase order, shipment, and timeline page.

File: src/views/Order.vue [hunk 2]
```diff
@@ -27,24 +27,19 @@
...
+            <ion-item lines="none" class="desktop-only">
+              <h2>{{ $t("Timeline") }}</h2>
+              <ion-note slot="end">1:07pm 6th Dec 2021</ion-note>
```
Comment (ymaheshwari1) [inline src/views/Order.vue:33]: `IonNote` component is not imported.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Comment by Dhiraj1405** [comment]
Not priority for now. Hence, closing this pull request.

**Pull request closed without merging**